### DISCUSSION TITLE
Configuring Lightly

### DISCUSSION THREAD

**Discussion by 0xVR** [Luwx/Lightly#43] [General]
Hi,

I know it's not meant for it but I'm using Lightly so that I can have transparent Dolphin on AwesomeWM (yeah I know, sounds like a mess lmao, I had to set dolphin to use breeze icons else they wouldn't load). Anyway, I was wondering if there's a way to configure Lightly either through the terminal or by editing some file? Since there's no option for it in qt5ct ofc

**Comment by Luwx** [comment]
Lightly has a config file named  `lightlyrc` located in ~/$HOME/.config. The parameters available can be found in https://github.com/Luwx/Lightly/blob/master/kstyle/lightly.kcfg

**Reply by 0xVR** [reply]
Perfect, thanks! Mind telling me where to find the full glass color scheme you used in the readme?

**Reply by Luwx** [reply]
You'll have to change the current color scheme (~/.local/share/color-schemes) for that ,  in the ```[Colors:Window]``` section, add a fourth value to ```BackgroundNormal```, that is the transparency. I don't know if this will work if you are not running plasma, because Lightly, like breeze, still depends a lot on kde stuff.

**Reply by 0xVR** [reply]
Yep, it didn't work lmao. Regardless, thanks for all the help!

**Comment by Langat-Dennis** [comment]
Can't seem to install it on kali Linux 2021.1
